### DISCUSSION TITLE
Change payment method order

### DISCUSSION THREAD

**Discussion by ginkel** [pretix/pretix#4204] [Q&A] [answered]
Hi there,

is there a way to change the order of payment methods during checkout so that SEPA direct debit shows up before bank transfer during checkout?

Thanks,
Thilo

**Comment by raphaelm** [comment, marked as answer]
No, it's alphabetical

**Reply by ginkel** [reply]
Thanks for the speedy reply! I figured out that I can override the name of the bank transfer option so that it is sorted after the direct debit variant.
